ExerciseThree.txt

```diff
@@ -0,0 +1,14 @@
+Test layer /src/test/java/hw4/ex1
+           /src/test/java/hw4/ex2
+Business layer ? (part of test package)
+Core layer /src/test/java/hw4/base
+           /src/main/java/hw4/pages
+           /src/main/java/hw4/composite
+           src/main/java/hw4/builder
+
+Test layer is implemented and located separately in the distinct package, but test steps maybe take out to another
+package in one java class. That would be part of the business layer.
+The core layer, including loading the driver, description of pages, are fully implemented.
+Configuration data (username and password) are stored in the property file.
```

Comment:
What additional info could be added to the properties file?

Reply:
Maybe information about the location of the test data, some log files. If that would be a web application, it contains setting variables such as port and server